tests/JMS/Serializer/Tests/Serializer/XmlSerializationTest.php

```diff
@@ -89,7 +89,7 @@ public function testPropertyIsCollectionOfObjectsWithAttributeAndValue()
 
     /**
      * @expectedException JMS\Serializer\Exception\InvalidArgumentException
-     * @expectedExceptionMessage The document type "<!DOCTYPE author [<!ENTITY foo SYSTEM "php://filter/read=convert.base64-encode/resource=XmlSerializationTest.php">]>" is not allowed. If it is safe, you may add it to the whitelist configuration.
+     * @expectedExceptionMessage The document type "<!ENTITY foo SYSTEM "php://filter/read=convert.base64-encode/resource=XmlSerializationTest.php">" is not allowed. If it is safe, you may add it to the whitelist configuration.
```

Comment:
Sorry for the late answer.
If I am not mistaken, there is `XmlSerializationTest::testExternalEntitiesAreDisabledByDefault` and `XmlSerializationTest::testDocumentTypesAreNotAllowed` to test that we can't deserialize without whitelisting [external]entities or doctypes.

As a test like the one you are mentionning, i didn't find any.
If you think it is important, maybe we can do it in an other PR after this one gets merged.